### ISSUE TITLE
CROSS JOIN not supported

### ISSUE THREAD

**Issue by BrandiATMuhkuh** [oguimbal/pgsql-ast-parser#49]
When I try to use a CROSS JOIN I'm getting the following errors

## `select * from a, b`

```
Error: Syntax error at line 2 col 16:

  select * from a,
                 ^
Unexpected comma token: ",". Instead, I was expecting to see one of the following:
```

## `select * from a CROSS JOIN b`

```
Error: Syntax error at line 3 col 17:

  select * from a CROSS
                  ^
Unexpected kw_cross token: "cross". Instead, I was expecting to see one of the following:
```

**Comment by oguimbal** [comment]
Hi,

Sorry for the delay, I dont have enough time to work on this these days.

I'll try to have a look when I have time.

**Comment by janpaepke** [comment]
I came across this issue as well and was preparing a bug report for pg-mem, when I figured out it's psql-ast-parser, which seems to be the origin.

I had already written most of it, so this might help debugging:

# Describe the bug

I stumbled across this error when debugging [slonik-typegen](https://github.com/mmkal/slonik-tools/tree/master/packages/typegen#readme) and was able to trace the origin to [pgsql-ast-parser](https://www.npmjs.com/package/pgsql-ast-parser).

I was able to reproduce it in the [pg-mem-playground](https://oguimbal.github.io/pg-mem-playground/).  
You can use the playground with the queries below to quickly reproduce.

It seems pgsql-ast-parser does not expect a from clause to contain multiple tables, even though this is explicitly allowed in the [PSQL Docs](https://www.postgresql.org/docs/9.5/sql-select.html#SQL-FROM):
> The FROM clause specifies **one or more source tables** for the SELECT. If multiple sources are specified, the result is the Cartesian product (cross join) of all the sources. But usually qualification conditions are added (via WHERE) to restrict the returned rows to a small subset of the Cartesian product.

This is the error thrown:
```
Syntax error at line 4 col 11:

      tableA,
            ^
Unexpected comma token: ",". Instead, I was expecting to see one of the following:

    - A "dot" token
    - A "dot" token
    - A "lparen" token
    - A "kw_as" token
    - A "kw_primary" token
    - A "kw_unique" token
    - A "word" token
    - A "kw_left" token
    - A "kw_right" token
    - A "kw_full" token
    - A "kw_inner" token
    - A "kw_where" token
    - A "kw_group" token
    - A "kw_join" token
    - A "kw_order" token
    - A "kw_limit" token
    - A "kw_offset" token
    - A "kw_fetch" token
    - A "kw_for" token
    - A "kw_union" token
    - A "semicolon" token
```

# To Reproduce

This query throws the error.
```sql
select
    tableA.col
from
    tableA, tableB
where
    tableA.fKey = tableB.id
```
As @BrandiATMuhkuh pointed out, the equivalent of the above query would be
```sql
select
    tableA.col
from
    tableA
cross join tableB
on tableA.fKey = tableB.id
```
This query also fails, while other joins seem to work as normal.

_Note:_ I did not include a table definition, as this is a parser error rather than a result error.

# Version
slonik-typegen relies on pgsql-ast-parser `^8.0.0`, but as it is also happening in the current pg-mem playground, which relies on `^9.3.2` I'm guessing this is a current issue.

**Comment by oguimbal** [comment]
Finally fixed in #69  ... huge thanks to @janpaepke
